### PR TITLE
Fix for issue# 233

### PR DESCRIPTION
**Pull request by msenmurugan**
Fix for koki selector match label is added by default during the creation of persistent volume claim

### CODE REVIEW

File: converter/converters/koki_rc_to_kube_v1_rc.go [hunk 1]
```diff
@@ -41,13 +41,6 @@ func Convert_Koki_ReplicationController_to_Kube_v1_ReplicationController(rc *typ
 		if len(kubeSpec.Template.Labels) == 0 {
```
Comment (rrati) [inline converter/converters/koki_rc_to_kube_v1_rc.go]: Update the comment above this if condition.  The code is no longer ensuring there is at least 1 label in the template and selector, but instead is ensuring that a label is set will match a selector if one is provided.

**Comment by wlan0** [comment]
This change breaks the API. Leaving this comment here to bump up the version and add this to release notes.

**Comment by wlan0** [comment]
@msenmurugan Can you make sure the CI passes? I'll merge once that's done

**Comment by msenmurugan** [comment]
@wlan0 In order to fix the CI build, I have commented the koki.selector labels in all testdata yamls. After this change, the functional test throws an error as "Missing pod spec". To fix the missing pod spec issue, I need to comment the below code in koki_rc_to_kube_v1_rc.go. After commenting the below code, all the issues are fixed except replication_controller_spec_without_template and replication_controllers/meta_test test scenario. 

```
func revertTemplate(kokiMeta *types.PodTemplateMeta, kokiSpec types.PodTemplate) (*v1.PodTemplateSpec, error) {
	var hasMeta = kokiMeta != nil
	var hasSpec = !reflect.DeepEqual(kokiSpec, types.PodTemplate{})
	if !hasMeta && !hasSpec {
		return nil, nil
	}
```

Can you please let me know whether the above check is required and also help me in fixing those two test scenarios?

File: converter/converters/koki_rc_to_kube_v1_rc.go (hunk 1 above)
Comment (msenmurugan) [reply converter/converters/koki_rc_to_kube_v1_rc.go]: Removed the comments

**Comment by wlan0** [comment]
LGTM

**Pull request merged**